pylint/checkers/base.py

```diff
@@ -2593,6 +2594,41 @@ def _check_type_x_is_y(self, node, left, operator, right):
         self.add_message("unidiomatic-typecheck", node=node)
 
 
+class ThreadingChecker(_BasicChecker):
+    """Checks for threading module
+
+    - useless with lock - locking used in wrong way that has no effect (with threading.Lock():)
+    """
+
+    msgs = {
+        "W9990": (  # TODO
```

Comment:
You can choose the id yourself it just need to not already exists and to not have the same beginning than another checker to not limit other checkers evolution ([2 first number](https://github.com/PyCQA/pylint/blob/6a7209b9b8b9bbd0fdf08ce22daf325c9d0d28ec/pylint/checkers/base_checker.py#L134)). This is definitely something we should document, I'll open an issue.